models/comments_model.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require("mongoose");
+
+const commentSchema = new mongoose.Schema({
+  userId: {
```

Comment:
change to `sender` to match post_model
```suggestion
  sender: {
```

make sure to update `request.rest` file as well

Reply:
254af7d0d768f2bc3afe57c7396f6cf92893bd2b